packages/app-cli/tests/MdToHtml.ts

```diff
@@ -43,7 +43,9 @@ describe('MdToHtml', function() {
 				bodyOnly: true,
 			};
 
-			if (mdFilename === 'checkbox_alternative.md') {
+			// render all checkbox tests with rendering type 2 for a
+			// cleaner output
+			if (mdFilename.includes('checkbox')) {
```

Comment:
Could you narrow the check down to just the concerned files (checkbox_alternative.md and the ones you've added)? Otherwise if someone creates a test that includes the string "checkbox", they might wonder why it behaves differently, so in general I prefer everything to be explicit in test units.